README.md

```diff
@@ -20,32 +20,37 @@ The user stories are categorized into 3 different groups: Must Have, Should Have
 
 Must Have: Missing such requirements will result in a non-viable solution and, as a result, failing the assessment.
 
-Should Have: Missing such requirements can be painful, but the solution is still viable. While it does not necessarily lead to failing the assessment, it is still devastating.
+Should Have: Missing such requirements can be painful, but the solution is still viable. While it does not necessarily lead to failing the assessment, it will be a penalty.
 
 Could Have: Such requirements are desirable but less important. This is a show-off zone where you can impress us.
 
 
 **Must Have**
-- As a player, I need to be able to know the hand when a new game starts.
-- As a player, I need to be able to end the game.
-- As a player, I need to be able to know the total number of points I have earned when the game ends.
+* As a player 
+  * I need to be able to see my hand when a new game starts.
+  * I need to be able to end the game.
+  * I need to be able to submit words constructed from my hand.
+  * I need to be able to know the total number of points I have earned when the game ends.
+  * I need to know if my word is invalid.
 
 **Should Have**
```

Comment:
I think we can put all **should have** into **could have**, because we now expect only 3 hours of work.

Reply:
I changed the wording so that 'should have' is less critical, because I think these stories are still more important than the ones in 'could have'.  Would you agree with that?

Reply:
Yeah, it sounds good! Thanks!